### ISSUE TITLE
Need help

### ISSUE THREAD

**Issue by hartotarigan** [Hacker0x01/hacker101#85]
Last login: Tue Jan 21 03:35:09 UTC 2020 on pts/2
Linux kali 3.18.31-fly-g22c0ebd-27271-geae3ddb #2 SMP PREEMPT Tue Jul 3 15:03:38 CST 2018 aarch64

The programs included with the Kali GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Kali GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
[root@kali][~]
 #cd hacker101
[root@kali][~/hacker101]
 #ls
404.html      README.md    assets        resourceGen.py  update_bootstrap.sh
CNAME         _config.yml  discord.html  resources       videos.md
Gemfile       _includes    favicon.ico   resources.md
Gemfile.lock  _layouts     index.md      resources.res
LICENSE       _sass        playlists     sessions
[root@kali][~/hacker101]
 #bundle exec jekyll serve
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /usr/share/rubygems-integration/all/specifications/erubis-2.7.0.gemspec:16.
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /usr/share/rubygems-integration/2.5.0/specifications/thin-1.7.2.gemspec:22.
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /usr/share/rubygems-integration/2.5.0/specifications/msgpack-1.1.0.gemspec:19.
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /usr/share/rubygems-integration/2.5.0/specifications/eventmachine-1.0.7.gemspec:21.
Configuration file: /root/hacker101/_config.yml
            Source: /root/hacker101
       Destination: /root/hacker101/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
WARNING: The `text-hide()` mixin has been deprecated as of v4.1.0. It will be removed entirely in v5.
         on line 10 of /root/hacker101/_sass/bootstrap/mixins/_text-hide.scss, in `text-hide'
         from line 57 of /root/hacker101/_sass/bootstrap/utilities/_text.scss
         from line 14 of /root/hacker101/_sass/bootstrap/_utilities.scss
         from line 41 of /root/hacker101/_sass/bootstrap/bootstrap.scss
         from line 2 of an unknown file

  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    Invalid US-ASCII character "\xE2" on line 5
jekyll 3.7.4 | Error:  Invalid US-ASCII character "\xE2" on line 5
[][root@kali][~/hacker101]
 #

**Comment by Willianvdv** [comment]
I'm not sure how I can help you, can you rephrase your question?

**Comment by hartotarigan** [comment]
Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
Invalid US-ASCII character "\xE2" on line 5
jekyll 3.7.4 | Error: Invalid US-ASCII character "\xE2" on line 5

**Comment by Prashles** [comment]
Please open a new issue if you still need help!
